README.md

```diff
@@ -5,5 +5,6 @@ A simple script to download your fastest, accepted solutions from Leetcode!
 ## Usage
 
 ```shell
- python3 fetch_submissions.py <username>
+ pip install requests
+ python fetch_submissions.py <username>
```

Comment:
Has to be `python3` because the script uses f-strings

Reply:
Good catch. I forgot to change this after removing the commands to create and activate a virtual environment as `python` linked to the Python3 location in the virtual environment.